Removed white stripes on LearnOpenGL hover element

CourtneyAbraham opened the pull request:
There are white stripes on the hover element on LearnOpenGL, this just sets the background image to none and opacity to 1.
this way it's actually possible to read what's there

Gusted commented:
Hi!

In which behavior does this hover element shows up? It seems to be hidden by default.

Regards,
Gusted

CourtneyAbraham commented:
[Here is a picture of the behaviour without what I've added](https://prnt.sc/12qp9yu)
[And here is a picture with what I've added enabled](https://prnt.sc/12qpcjj)

Both times I am hovering over the region I've highlighted red

Gusted commented on src/config/dynamic-theme-fixes.config:
```diff
@@ -6274,6 +6274,16 @@ INVERT
+CSS
+#hover {
+    background-image: none !important;
+    opacity: 1;
```
```suggestion
    opacity: 1 !important;
```
